demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1404,9 +1404,10 @@ def format_file_path(self, file_path, old_path, old_format_files):
         else:
             return file_path
 
-    def ignore_test_doc_non_pack_file(self, file_path: str) -> bool:
+    def ignore_files_irrelevant_for_validation(self, file_path: str) -> bool:
         # ignore doc data, test_data and non pack files
-        if any(test_dir in str(file_path) for test_dir in TESTS_AND_DOC_DIRECTORIES) or PACKS_DIR not in file_path:
+        if any(name in str(file_path) for name in VALIDATION_USING_GIT_IGNORABLE_DATA) \
+                or PACKS_DIR not in file_path:
```

Comment:
I would replace the order of the conditions (it is better performance-wise)

Reply:
Fixed